packages/next/src/server/stream-utils/node-web-streams-helper.ts

```diff
@@ -123,40 +123,57 @@ export async function streamToString(
   return string
 }
 
-export function createBufferedTransformStream(): TransformStream<
-  Uint8Array,
-  Uint8Array
-> {
+export type BufferedTransformOptions = {
+  /**
+   * Flush synchronously once the buffer reaches this many bytes.
+   */
+  readonly maxBufferByteLength?: number
+}
+
+export function createBufferedTransformStream(
```

Comment:
Are we confident with changing streams that are already used by other functionality? Seems risky to not use a dedicated stream factory.